### DISCUSSION TITLE
Create ovf template from Azure marketplace windows10 image

### DISCUSSION THREAD

**Discussion by shettys-ops** [vmware/packer-examples-for-vsphere#641] [Help] [answered]
can you share an example where we can use Packer, Source image is Azure Marketplace and then export it to ovf template, which can then be uploaded to Vmware Content library

**Comment by tenthirtyam** [comment, marked as answer]
This would be out of scope for this project. I recommend using discuss.hashicorp.com under the packer/vsphere topic or the [unofficial Hashicorp Community Discord](https://dev.to/angelonfira/discord-community-for-hashicorp-products-43gk).
